dlp/src/main/java/dlp/snippets/DeIdentifyWithDateShift.java

```diff
@@ -98,7 +98,7 @@ public static void deIdentifyWithDateShift(
       // Combine configurations into a request for the service.
       DeidentifyContentRequest request =
           DeidentifyContentRequest.newBuilder()
-              .setParent(ProjectName.of(projectId).toString())
+              .setParent(String.format("%s/locations/global", ProjectName.of(projectId).toString()))
```

Comment:
Really seems like the client library should be changed, like adding a `toGlobalString()` or `asParent()` methods.

Reply:
Unfortunately the client library is generated, so we don't currently have a straightforward way to add arbitrary helper functions. 

I agree that it's not ideal, but I'm unaware of better available options.

Reply:
FYI, the Python library has a class method `location_path`. Worth checking if there's a similar thing in Java client.

Reply:
Thanks Takashi, you're right! In Java it's LocationName and it's definitely the right way to go. Updating now.

Reply:
@sushicw Please ping me once you've updated the PR.